lib/liquid/tags/if.rb

```diff
@@ -84,14 +84,21 @@ def lax_parse(markup)
     def strict_parse(markup)
       p = Parser.new(markup)
 
-      condition = parse_comparison(p)
+      conditions = [parse_comparison(p)]
```

Comment:
Not ideal to have to build this stack, but without recursion (implicit stack) I dunno how to implement this fix.